src/style.css

```diff
@@ -0,0 +1,116 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+.bigForm {
+  box-sizing: border-box;
+  position: relative;
+  margin: 20px 0;
+}
+
+.input {
+  /* position: relative; */
+  background-color: #fff;
+  box-sizing: border-box;
+  border: solid 1px rgba(225, 231, 237, 1);
+  border-radius: 4px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  color: rgba(61, 78, 97, 1);
+  font-size: 16px;
+  height: 70px;
+  width: 100%;
+  padding: 24px 0 25px 62px;
+  font-family: Avenir, sans-serif;
+  outline: none;
+}
+
+.smallFrom {
+  box-sizing: border-box;
+  position: relative;
+  margin: 20px 0;
+}
+
+.input-small {
+  background-color: #fff;
+  box-sizing: border-box;
+  border: solid 1px rgba(225, 231, 237, 1);
+  border-radius: 4px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  color: rgba(61, 78, 97, 1);
+  font-size: 14px;
+  width: 100%;
+  font-family: Avenir, sans-serif;
+  padding: 1px 0 0 33px;
+  outline: none;
+  height: 42px;
+}
+
+.icon {
+  position: absolute;
+  display: block;
+  width: 19px;
+  height: 19px;
+  top: 25px;
+  left: 26px;
+}
+
+.icon--small {
+  display: block;
+  position: absolute;
+  width: 11px;
+  height: 11px;
+  top: 15px;
+  left: 13px;
+}
+
+.input-small:hover {
+  box-shadow: 0 4px 4px rgba(0, 0, 0, 0.25);
+  font-weight: 300;
+  text-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  outline: none;
+
+}
+
+.input:hover {
+  border-radius: 4px;
+  box-shadow: 0 4px 4px rgba(0, 0, 0, 0.25);
+  outline: none;
+  font-weight: 300;
+
+}
```

Comment:
Same. You could use one common style and modifiers. For example `input input--small`, `input input--big`